<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>EditClientDialog</class>
    <widget class="QDialog" name="EditClientDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Edit Client</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="label_name">
                    <property name="text">
                        <string>Name:</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="nameLineEdit"/>
            </item>
            <item>
                <widget class="QLabel" name="label_surname">
                    <property name="text">
                        <string>Surname:</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="surnameLineEdit"/>
            </item>
            <item>
                <widget class="QLabel" name="label_family">
                    <property name="text">
                        <string>Family:</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QComboBox" name="familyComboBox">
                    <property name="editable">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>EditClientDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>EditClientDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>